Database/Corrections/sodQuestFixes.lua

```diff
@@ -2502,6 +2502,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -416091,
             [questKeys.zoneOrSort] = sortKeys.HUNTER,
         },
+        [90151] = {
+            [questKeys.name] = "Sniper Training",
+            [questKeys.startedBy] = {{210549}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 15,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Defias Scouts until the rune drops."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Spot a Defias Scouts with Hunter's Mark and defeat them until the rune drops."},
```

Reply:
No
Sentence malformed, reads wrong
"Find and mark Defias Scouts with Hunter's Mark, kill them while marked till the rune drops"

The above reads better

Reply:
Good point, thanks!

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5453/commits/e1056afed1e95c1bed92e9912ea3adcbf6494aa6